packages/react-components/react-tag-picker-preview/src/components/TagPickerOptionGroup/TagPickerOptionGroup.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import type { ForwardRefComponent } from '@fluentui/react-utilities';
+import { useTagPickerOptionGroup } from './useTagPickerOptionGroup';
+import { renderTagPickerOptionGroup } from './renderTagPickerOptionGroup';
+import { useTagPickerOptionGroupStyles } from './useTagPickerOptionGroupStyles.styles';
+import type { TagPickerOptionGroupProps } from './TagPickerOptionGroup.types';
+
+/**
+ * TagPickerOptionGroup component - TODO: add more docs
+ */
+export const TagPickerOptionGroup: ForwardRefComponent<TagPickerOptionGroupProps> = React.forwardRef((props, ref) => {
+  const state = useTagPickerOptionGroup(props, ref);
```

Comment:
`useTagPickerOptionGroup` and `renderTagPickerOptionGroup` are just direct proxies to  the combobox equivalent. You can keep the types and simply call the original functions here to avoid these proxies